nix/formatting.nix

```diff
@@ -1,22 +0,0 @@
-{ writeShellScriptBinInRepoRoot
-, pkgs
-}:
-let
```

Comment:
@zeme-wana, it looks like IOGX handles the formatting now so we don't need this.